Makefile

```diff
@@ -1,6 +1,6 @@
 RELEASE_REPO ?= public.ecr.aws/karpenter
 RELEASE_VERSION ?= $(shell git describe --tags --always)
-RELEASE_MANIFEST = releases/$(CLOUD_PROVIDER)/manifest.yaml
+RELEASE_CHART = charts
```

Comment:
Any reason to make this dynamic? What is the usecase for changing this variable?

Reply:
Nope, I was dynamic just to mimic the behaviour of RELEASE_MANIFEST variable. But RELEASE_MANIFEST had a dependency for CLOUD_PROVIDER (which is not the case of RELEASE_CHART).

Agreed